src/JuMP.jl

```diff
@@ -481,13 +481,26 @@ if VERSION >= v"0.7-"
     Base.broadcastable(cref::ConstraintRef) = Ref(cref)
 end
 
-# TODO: should model be a parameter here?
-function MOI.delete!(m::Model, cr::ConstraintRef{Model})
-    @assert m === cr.m
-    MOI.delete!(m.moi_backend, index(cr))
+"""
+    delete(model::Model, constraint_ref::ConstraintRef)
+
+Delete the constraint associated with `constraint_ref` from the model `model`.
+"""
+function delete(model::Model, constraint_ref::ConstraintRef{Model})
+    # TODO: should model be a parameter in the constraint_ref?
```

Comment:
`Model` is already a parameter in the constraint ref. Not sure what this TODO means, maybe just delete?